src/pages/api/deleteList.ts

```diff
@@ -1,6 +1,6 @@
 import { NextApiRequest, NextApiResponse } from "next";
 import { z, ZodError } from "zod";
-import { Items, List, Prisma, PrismaClient } from "@prisma/client";
+import { PrismaClient } from "@prisma/client";
 
 const prisma = new PrismaClient();
```

Comment:
this needs to import the exported prisma client in the prisma.ts file, and not instantiate a new prisma client here

Reply:
applied